{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 21:00:14"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 21:00:14"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 21:00:14"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 21:00:24"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 21:00:24"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 21:00:24"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 21:01:27"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 21:01:27"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 21:01:27"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 21:06:31"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 21:06:31"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 21:06:31"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 21:07:42"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 21:07:42"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 21:07:42"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 21:08:49"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 21:08:49"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 21:08:49"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 21:09:29"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 21:09:29"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 21:09:29"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 21:09:38"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 21:09:38"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 21:09:38"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 21:09:47"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 21:09:47"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 21:09:47"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 21:09:59"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 21:09:59"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 21:09:59"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 21:10:32"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 21:10:32"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 21:10:32"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 21:10:57"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 21:10:57"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 21:10:57"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 21:11:16"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 21:11:16"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 21:11:16"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 21:11:23"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 21:11:23"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 21:11:23"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 21:12:28"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 21:12:28"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 21:12:28"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 21:12:39"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 21:12:39"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 21:12:39"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 21:13:54"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 21:13:54"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 21:13:54"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 21:14:23"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 21:14:23"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 21:14:23"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 21:14:55"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 21:14:55"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 21:14:55"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 21:15:21"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 21:15:21"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 21:15:21"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 21:15:38"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 21:15:38"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 21:15:38"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 21:16:46"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 21:16:46"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 21:16:46"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 21:17:08"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 21:17:08"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 21:17:08"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-15 21:17:46"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-15 21:17:46"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-15 21:17:46"}
